src/main/flight/pid.c

```diff
@@ -225,21 +225,21 @@ void pidInitFilters(const pidProfile_t *pidProfile)
         default:
             dtermLpfApplyFn = nullFilterApply;
             break;
-        case FILTER_PT1:
```

Comment:
(dterm and stage2 mixed here)

Reply:
FILTER_PT1 became STAGE2_PT1, that's the confusion I don't like about new code and currently thinking about unifying all lowpass filters.